recipes/skore-local-project/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
The license file is already included in the source distribution, so you should remove this copy